sgkit/tests/test_ld.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+from sgkit.stats.ld import rogers_huff_r2_between, rogers_huff_r_between
+
+
```

Comment:
Good to compare with scikit allele on a range of simulated datasets.

Reply:
+1